docs/testing-other-systems.rst

```diff
@@ -1,7 +1,7 @@
 .. _testing-other-systems:
 
 ========================
-Testing non-HTTP systems 
```

Comment:
Maybe split this into two sections? ”Testing non-HTTP systems” and ”Testing requests-based SDKs”?